cognite/client/_api/sequences.py

```diff
@@ -185,6 +186,74 @@ def list(
         ).dump(camel_case=True)
         return self._list(method="POST", filter=filter, limit=limit)
 
+    def _dataframe_to_columns_definition(self, dataframe, allow_integers=True):
+        np, pd = utils._auxiliary.local_import("numpy", "pandas")
+
+        def map_type(t):
+            if allow_integers and np.issubdtype(t, np.integer):
+                return "LONG"
+            elif np.issubdtype(t, np.number):
+                return "DOUBLE"
+            else:
+                return "STRING"
+
+        return [{"externalId": col, "name": col, "valueType": map_type(t)} for col, t in dataframe.dtypes.iteritems()]
+
+    def create_from_dataframe(
```

Comment:
Would it be better to piggyback on insert_dataframe()?

i.e. adding a `create_sequence_if_missing` parameter to SequencesDataApi.insert_dataframe().

Reply:
interesting idea, I'm not sure.